templates/projects/tasks_browse.webpack.ejs

```diff
@@ -247,9 +247,23 @@
                 {% endif %}
                 {% if 'gold_task' in filter_data.display_columns %}
                     {% if t.calibration %}
-                    <td style="width: 100%"><a class="label label-warning" style="display: block; margin: 0 0.25rem" target="_blank" href="{{ url_for('project.task_presenter', short_name=project.short_name, task_id=t.id) + '?mode=gold' }}">Update</a></td>
+                        <td style="width: 100%">
+                            <!-- Allow project owners and admins to update Gold tasks -->
+                            {% if current_user.id in project.owners_ids or current_user.admin %}
```

Comment:
I think this will let regular co-owner as well to update whereas only subadmin coowners to be allowed to make such update; same applies for make gold option below.

Reply:
That is correct, co-owners can also update. The error does not occur when a co-owner updates a gold task - the original [error](https://jira4.prod.bloomberg.com/browse/RDISCROWD-1635?focusedCommentId=359983&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-359983) was only under the condition of a sub-admin modifying a gold task. Wouldn't this be the desired behavior?

Reply:
Fixed via https://github.com/bloomberg/pybossa-default-theme/pull/122/commits/1e5af2b362d1351cb5eba653defe0ee558911508